src/main/java/edu/kis/powp/jobs2d/drivers/adapter/ModifiableLineAdapter.java

```diff
@@ -0,0 +1,27 @@
+package edu.kis.powp.jobs2d.drivers.adapter;
+
+import edu.kis.legacy.drawer.shape.line.AbstractLine;
+import edu.kis.powp.jobs2d.drivers.IModifiableLine;
+
+import java.awt.*;
+
+public class ModifiableLineAdapter extends AbstractLine implements IModifiableLine
+{
+    @Override
+    public void setColor(Color color)
+    {
+        this.color = color;
+    }
+
+    @Override
+    public void setThickness(float thickness)
+    {
+        this.thickness = thickness;
+    }
+
+    @Override
+    public void setDotted(boolean dotted)
+    {
+        this.dotted = dotted;
+    }
```

Comment:
is it tested anywhere?!

Reply:
Yes, in TestJobs2dPatterns class.

Reply:
No. This is the last commit in PR.

Reply:
Right, i forgot to push my last changes. Can i gain some extra points by preparing another PR with some issues fixed ?

Reply:
Yes, but not much.